src/EventListener/SendEventOnRetriedMessage.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SymfonyCasts\MessengerMonitorBundle\EventListener;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Messenger\Event\WorkerMessageFailedEvent;
+use Symfony\Component\Messenger\EventListener\SendFailedMessageForRetryListener;
+use Symfony\Component\Messenger\Stamp\SentToFailureTransportStamp;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use SymfonyCasts\MessengerMonitorBundle\FailedMessage\MessageRetriedByUserEvent;
+use SymfonyCasts\MessengerMonitorBundle\Stamp\MonitorIdStamp;
+
+/**
+ * @internal
```

Comment:
I would be cool with some description up here - it could help us follow the logic later